README.md

```diff
@@ -17,9 +17,11 @@ Features:
 
 How developers use it:
 
-1. (preferred) This ruleset provides an Aspect CLI plugin,
+1. Run with vanilla Bazel, by placing a couple commands in a shell script, Makefile, or similar wrapper.
+2. (preferred) This ruleset provides an Aspect CLI plugin,
```

Comment:
Why move the preferred solution last?

Reply:
It's the order I recorded the demo, but yeah it can go back